installer/scripts/preinstall.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#This script check for installed python interpreter with opropriate version.
+# In some distributions, there is no 'python' command even if python interpreter is installed.
+# In this cases script have to exit with an error.
+# This is important because all agent scripts rely on '/usr/bin/env python' command.
+
+
+echo "Check for python."
+# if command was not successful, print hint message and exit.
+if ! /usr/bin/env python --version ; then
+    echo -e "\e[31mCommand 'python' is not found.\e[0m"
+    echo "You can install it by running command:"
+    # get 'ID_LIKE' and 'ID' fields from '/etc/os-release' file and then search for distributions key words.
+    if [[ -f "/etc/os-release" ]]; then
+        found_distros=$(grep -qE  "^ID_LIKE=|^ID=" /etc/os-release)
```

Comment:
Hm, are you sure ``-q`` flag needs to be there? Since if it is, it will run in quite mode and won't output anything.

```bash
$ grep -qE  "^ID_LIKE=|^ID=" /etc/os-release
$ grep -E  "^ID_LIKE=|^ID=" /etc/os-release
ID=fedora
```

Reply:
On that note, let's please add some test cases for that in the future once other smoke tests PR is merged.

Reply:
Yes, my bad, thanks for noticing that.

Reply:
> On that note, let's please add some test cases for that in the future once other smoke tests PR is merged.

Sure thing.

Reply:
And as discussed before, I'm fine with just mocking ``/etc/os-release`` file in those tests.

Eventually we do want tests to run on more of the supported distros, but this would be too big of a task for now.